rdrf/rdrf/auth/ldap_backend.py

```diff
@@ -13,25 +14,35 @@ class RDRFLDAPBackend(LDAPBackend):
     }
 
     def get_or_build_user(self, username, ldap_user):
-        logger.debug("in get_or_build_user")
         user, built = super().get_or_build_user(username, ldap_user)
-        if built:
-            logger.debug("in built clause")
-            from rdrf.models.definition import Registry
-            from registry.patients.models import WorkingGroup
-            registry_model = Registry.objects.get(code="ICHOMCRC")
-            wg = WorkingGroup.objects.get(registry=registry_model,
-                                          name="RPH")
-            user.is_active = True
-            logger.debug("set is_active")
+
+        # We need to update the user information whatever it existed or not in RDRF database.
+
+        from rdrf.models.definition.models import Registry
+        from registry.groups.models import WorkingGroup
+        registry_model = Registry.objects.get(code=settings.RDRF_AUTH_LDAP_REGISTRY_MODEL)
+        wg = WorkingGroup.objects.get(registry=registry_model,
+                                      name=settings.RDRF_AUTH_LDAP_WORKING_GROUP)
+
+        # Check if user must be staff anyway
+        if settings.RDRF_AUTH_LDAP_USERS_MUST_BE_STAFF:
```

Comment:
All users must be staff so I think  the setting is redundant - being staff is forced.

Reply:
Good point. In this case I will remove the staff group binding too. Actually it is better this way, if ever we need these settings we can add them back. But it is unlikely anyway.